New version: FlxQTL v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: FlxQTL
- Repository: https://github.com/senresearch/FlxQTL.jl
- Created by: @hkim89
- Version: v1.0.0
- Commit: a4555a3e2e9bc079088ff9eff449809e45756249
- Reviewed by: @hkim89
- Reference: https://github.com/senresearch/FlxQTL.jl/commit/a4555a3e2e9bc079088ff9eff449809e45756249#commitcomment-151775164
- Description: A comprehensive QTL analysis tool by multivariate linear mixed  model.
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- Version upgrade to v. 1.0.0 for numerical stability

## Breaking changes
- All existing scan functions and permutations have changed by running penalized log likelihood functions
- New functions are included: `gene1Scan`, `gene2Scan` for pre-estimating a `Kc` matrix automatically inside, and `getKc` for  separately precomputing `Kc`.  
- New keyword arguments for Inverse-Wishart prior distribution information are included, e.g. df_prior, Prior
- Pkg documentation is updated accordingly
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd60bb8f1d3bc10efe09f2e81ae0a188d17bd6d3326b1fa63d7765b885e1a672500811b78fdb31a5993380afaccdf187286a8f21b34ad02b815f8e09fd0983956af3dd0e25ac30af0744e80b67f4c370c96f7370736014accc6ae7f1b6faeaced44f8ef878818b27ccc902b0e179b8fbdf57cfc3ebd19cfce38b2c7df331284a2dc65148d9684e13a8cda3331c3c0129bc51c592293831b22e76813f41d87586a1d4088bfd39d07bf59a1e78a461e2089db -->

hkim89 commented:
@JuliaRegistrator register

Release notes:
- Version upgrade to v. 1.0.0 for numerical stability

## Breaking changes
- All existing scan functions and permutations have changed by running penalized log likelihood functions
- New functions are included: `gene1Scan`, `gene2Scan` for pre-estimating a `Kc` matrix automatically inside, and `getKc` for  separately precomputing `Kc`.  
- New keyword arguments for Inverse-Wishart prior distribution information are included, e.g. df_prior, Prior
- Pkg documentation is updated accordingly

JuliaRegistrator committed:
New version: FlxQTL v1.0.0

UUID: ede81e08-c6d8-4fe3-94c2-f928d9d678ed
Repo: https://github.com/senresearch/FlxQTL.jl.git
Tree: 2ec14176d7ea628287a2d61021a550bb079b7d11

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
